setup.py

```diff
@@ -10,7 +10,8 @@
     packages=find_packages(),
     license='ISC',
     install_requires=[
-        'ruamel.yaml'
+        'ruamel.yaml',
+        'rich~=13.0'
```

Comment:
>  when semver specifically states that those should only be incremented when incompatible API changes are made. 

You assume 1) the world is rational and 2) actually follows semver, rather than their own emotional interpretation of it.

Neither happens in practice.